apps/src/storage/firebaseStorage.js

```diff
@@ -1241,9 +1241,13 @@ FirebaseStorage.getColumn = function (
     tableName,
     {},
     records => {
-      let columnValues = [];
-      records.forEach(row => columnValues.push(row[columnName]));
-      onSuccess(columnValues);
+      if (records === null) {
+        onSuccess(null);
```

Comment:
null check here avoids exception on null.forEach and passes null back to handleGetColumn here: https://github.com/code-dot-org/code-dot-org/blob/cassi/applab-getcol-onfailure-fix/apps/src/applab/commands.js#L1877-L1878